packages/webhooks/model-validator/test.schema.ts

```diff
@@ -0,0 +1,88 @@
+export type Maybe<T> = T | null
+export type Scalars = {
+    ID: string;
+    String: string;
+    Boolean: boolean;
+    Int: number;
+    Float: number;
+    JSON: any;
+    Upload: any;
+}
+export type File = {
+    __typename?: 'File';
+    id?: Maybe<Scalars['ID']>;
+    publicUrl?: Maybe<Scalars['String']>;
+}
+export type B2BAppAccessRight = {
```

Comment:
Is that types hard coded here?

Reply:
It's "very-very-minified" schema for test purposes only....I've just took random model from condo schema.... 😄

Reply:
I've also manually removed some fields, since it's not necessary....(why to keep all string fields, if logic is same for all of them)

Reply:
Made test schema more abstract :)